frontend/templates/product-list/server.tsx

```diff
@@ -207,6 +208,19 @@ export const getProductListServerSideProps = ({
          };
       }
 
+      if (productList.redirectTo) {
+         const destination = productListPath({
+            productList: productList.redirectTo,
+            itemType: itemType ?? undefined,
+         });
```

Comment:
Not sure how easy this is but it would be nice to forward any tracking params (i.e. utm etc params).